src/app/components/category-card/category-card.component.scss

```diff
@@ -0,0 +1,125 @@
+.flex-center {
+    display: flex;
+    justify-content: center;
+  }
+  .card {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    transition: 0.3s;
+    height: 95%;
+    width: 60%;
+    margin: auto;
+  }
+  .card:hover {
+    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  }
+  .mat-form-field {
+    width: 100%;
+  }
+  .buttonCat {
+    border-radius: 15px;
+    border: none;
+    color: white;
+    background-color: #E92B8E;
+    height: 50px;
+    width: 50px;
+    margin: 6px;
+  }
```

Comment:
If it hasn't been done yet, you should define a class for buttons in the style.css file. Except from the background-color, the other properties and values are the same, so you are repeating yourself.